x-pack/docs/en/security/operator-privileges/operator-only-functionality.asciidoc

```diff
@@ -12,12 +14,14 @@ They may evolve in future releases, but are otherwise fixed in a given {es} vers
 * <<voting-config-exclusions,Voting exclusions APIs>>
 * <<delete-license,Delete license API>>
 * <<update-license,Update license API>>
-* <<autoscaling-apis,Autoscaling APIs>>
+* <<autoscaling-put-autoscaling-policy,Put autoscaling policy API>>
+* <<autoscaling-delete-autoscaling-policy,Delete autoscaling policy>>
```

Comment:
This is missing "API"
```suggestion
* <<autoscaling-delete-autoscaling-policy,Delete autoscaling policy API>>
```
But won't you get the right outcome just by relying on asciidoctor's auto link text?
```suggestion
* <<autoscaling-delete-autoscaling-policy>>
```

Reply:
Thanks that will do. I copy/pasted from https://raw.githubusercontent.com/elastic/elasticsearch/7.12/docs/reference/autoscaling/apis/autoscaling-apis.asciidoc